daemon/src/GHCSpecter/UI/ConcurReplica/DOM.hs

```diff
@@ -0,0 +1,12 @@
+module GHCSpecter.UI.ConcurReplica.DOM
+  ( text,
+  )
+where
+
+import Concur.Core (Widget, display)
+import Data.Text (Text)
+import GHCSpecter.UI.ConcurReplica.Types (IHTML (..), embed)
+import Replica.VDOM (VDOM (..))
+
+text :: Text -> Widget IHTML a
```

Comment:
Concur.Replica.DOM.text was specialized to Widget HTML, so I reintroduced this text for Widget IHTML. Fortunately, all the other DOM building primitives like `div` were parameterized over `m` with the WidgetConstraints constraint.

Reply:
added comment.